game/atmosphere.rpy

```diff
@@ -1,8 +1,24 @@
-# Sky types for the classroom
-image sky_day overcast = "mod_assets/backgrounds/classroom/sky_day_overcast.png"
-image sky_day rain = "mod_assets/backgrounds/classroom/sky_day_rain.png"
-image sky_day sunny = "mod_assets/backgrounds/classroom/sky_day_sunny.png"
-image sky_day thunder = "mod_assets/backgrounds/classroom/sky_day_thunder.png"
+# Weather data
+default persistent.jn_weather_api_key = None
```

Comment:
We should keep the `_jn` prefix for all persist vars I think. Want to eventially migrate all to be consistent as this is semi-private.
```suggestion
default persistent._jn_weather_api_key = None
```